Resources/Prototypes/_NF/Entities/Objects/Specific/chemical-containers.yml

```
@@ -303,6 +303,41 @@
   - type: StaticPrice
     vendPrice: 0
 
+- type: entity
+  parent: BaseItem
```

```suggestion
  parent: [BaseC3CultContrabandNoValue, BaseItem]
```
use cult if it's supposed to be a cult item, otherwise just use C3